cc/modules/ximgproc/SuperpixelLSC.cc

```diff
@@ -0,0 +1,62 @@
+#ifdef HAVE_XIMGPROC
+
```

Comment:
```
#include <opencv2/core.hpp>
#if OPENCV_MINOR_VERSION > 0
```

probably have to include opencv core here, otherwise OPENCV_MINOR_VERSION will be unkown in this file